client/src/main/java/com/arcbees/beestore/client/application/widget/sidepanel/checkout/AddressPresenter.java

```diff
@@ -66,12 +65,13 @@ public void onPaymentDetailsUpdated() {
     private boolean validateContactInfo() {
         ContactInfoDto contactInfo = getContactInfo();
 
-        return !Strings.isNullOrEmpty(contactInfo.getFirstName()) &&
+/*        return !Strings.isNullOrEmpty(contactInfo.getFirstName()) &&
```

Comment:
Yes, I wanted to disabled the contact information validation until development was complete, but I'll just remove the comment for now.